src/ExpressiveInstaller/Resources/src/ExpressiveAuraConfig.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace App;
+
+use ArrayObject;
+use Aura\Di\Container;
+use Aura\Di\ContainerConfigInterface;
+
+/**
+ * Configuration for the Aura.Di container.
+ *
+ * This class provides functionality for the following service types:
+ *
+ * - Aliases
+ * - Delegators
+ * - Factories
+ * - Invokable classes
+ * - Services (known instances)
+ */
+class ExpressiveAuraConfig implements ContainerConfigInterface
```

Comment:
@weierophinney I see that you created a layer that makes all supported containers compatible with SM-like config. What about moving this layer to a "expressive-container-interop" package? I think that this is usable as well by other frameworks :)